WordPress/src/test/java/org/wordpress/android/ui/jetpack/backup/download/BackupDownloadViewModelTest.kt

```diff
@@ -34,13 +164,56 @@ class BackupDownloadViewModelTest {
         assertThat(toolbarStates.size).isEqualTo(0)
     }
 
+    @Test
+    fun `backupDownloadState is writtenToBundle`() {
+        viewModel.start(savedInstanceState = null)
+
+        viewModel.writeToBundle(savedInstanceState)
+        verify(savedInstanceState)
+                .putParcelable(any(), argThat { this is BackupDownloadState })
+    }
+
+    @Test
+    fun `when setToolbarState is invoked, toolbar state is updated`() {
+        val toolbarStates = initObservers().toolbarState
+
+        viewModel.start(null)
+        viewModel.setToolbarState(DetailsToolbarState())
+        assertThat(toolbarStates.last()).isInstanceOf(DetailsToolbarState::class.java)
+
+        viewModel.setToolbarState(ProgressToolbarState())
+        assertThat(toolbarStates.last()).isInstanceOf(ProgressToolbarState::class.java)
+
+        viewModel.setToolbarState(CompleteToolbarState())
+        assertThat(toolbarStates.last()).isInstanceOf(CompleteToolbarState::class.java)
+    }
+
+    @Test
+    fun `step index is restored`() {
+        val index = 2
+        whenever(savedInstanceState.getInt(KEY_BACKUP_DOWNLOAD_CURRENT_STEP)).thenReturn(index)
+        whenever(savedInstanceState.getParcelable<BackupDownloadState>(KEY_BACKUP_DOWNLOAD_STATE))
+                .thenReturn(backupDownloadState)
+        viewModel.start(savedInstanceState = savedInstanceState)
```

Comment:
Minor (🔍): Consider removing the explicit `savedInstanceState` named argument. Within this whole test suite, I am seeing both, this to be and not being used, most of the tests are not using it. This way all test will end up being more consistent. Also, due to the fact that this function accepts only one parameter it makes it less necessary.